src/style.css

```diff
@@ -1,3 +1,87 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --color_hover: #00acdc;
+  --color_text: #000;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500px;
+}
+
 body {
   margin: 0;
 }
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  color: var(--color_hover);
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_item {
+  list-style: none;
+  margin: 0 10px;
+  text-transform: uppercase;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.nav__list li:first-child {
+  margin-left: 0;
+}
+
+.nav__list li:last-child {
```

Comment:
Instead of selecting 'li' elements, use the defined class .nav_item to target the desired elements.